examples/troopjs_require/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "private": true,
+  "dependencies": {
+    "jquery": "^2",
+    "when": "^3.4",
+    "poly": "git://github.com/cujojs/poly#0.6.0",
+    "requirejs": "^2.1",
+    "todomvc-app-css": "^2",
```

Comment:
unless we change all the tests... I would rather wait to land this change only increasing the spread of implementations :grimacing: